### ISSUE TITLE
Dark Mode

### ISSUE THREAD

**Issue by fszontagh** [fszontagh/sd.cpp.gui.wx#4]

**Comment by fszontagh** [comment]
DarkMode is not available in the current version of wxWidgets. Some patch is available in the main branch with lot of missing features

**Comment by Amin456789** [comment]
honestly the light mode u got is very good on eyes as the windows is small and i love it so much, please work on  other important stuff and let this one go if it is too hard to do

**Comment by Amin456789** [comment]
great to hear! thank u for ur hard work

**Comment by iwr-redmond** [comment]
You may wish to revisit Dark Mode support after the next release, as version 3.2.6 of wxWidgets [included](https://github.com/wxWidgets/wxWidgets/blob/v3.2.6/docs/changes.txt#L265) a range of fixes, and there has been further work since (e.g. [fbc76d6](https://github.com/wxWidgets/wxWidgets/commit/fbc76d64fae12811ebf441093c664122cd1f809c), [cdf8d0a](https://github.com/wxWidgets/wxWidgets/commit/cdf8d0a0fec81995def926132e093d29266d28e9), [ec2f048](https://github.com/wxWidgets/wxWidgets/commit/ec2f0487f1473872da7ebbf46f897ba6b5cbac85)).

**Comment by fszontagh** [comment]
Yeah, my hope is  in the version 3.3.0, but as i know so there is no release date: 
https://www.wxwidgets.org/blog/2024/10/hello-darkness/

I tried earlier with the v3.3.0 to see how is going the dark mode on win, but it was then really ugly. 

There is an envirnonment variable to set the darkmode on win (in cmd, not powershell):

`set WX_MSW_DARK_MODE=1 && .\stablediffusiongui.exe `
_Where 1 means to use system mode, 2 means force to dark mode._ 

But it's not working with 3.2.6 of course. 

If they release the v3.3, i will upgrade to it.

**Comment by iwr-redmond** [comment]
Version 3.3.0 has [arrived](https://github.com/wxWidgets/wxWidgets/releases/tag/v3.3.0).

**Comment by iwr-redmond** [comment]
> Now it's hard to follow the changes in the sd.cpp lib, because in one time there is too many and undocumented changes.

Many of the changes were discussed at length in their respective PRs, e.g. [stable-diffusion.cpp#696](https://github.com/leejet/stable-diffusion.cpp/pull/696), [stable-diffusion.cpp#707](https://github.com/leejet/stable-diffusion.cpp/pull/707), [stable-diffusion.cpp#744](https://github.com/leejet/stable-diffusion.cpp/pull/744), etc.
